### PR TITLE
[i18n-vi] Translate README.md to Vietnamese

### PR DESCRIPTION
**Pull request by hoangsvit**
# What does this PR do?

This PR adds a Vietnamese translation for the README.md.

## Before submitting

- [ ]  This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

## Who can review?

Documentation: @stevhliu and @MKhalusova

### CODE REVIEW

File: README_vi.md [hunk 1]
```diff
@@ -0,0 +1,579 @@
...
+* 🗣️ Âm thanh, cho các nhiệm vụ như nhận dạng giọng nói và phân loại âm thanh.
+
+Các mô hình Transformer cũng có thể thực hiện các nhiệm vụ trên **nhiều modalities kết hợp**, như trả lời câu hỏi về bảng, nhận dạng ký tự quang học, trích xuất thông tin từ tài liệu quét, phân loại video và trả lời câu hỏi hình ảnh.
+
+🤗 Transformers cung cấp các API để tải xuống và sử dụng nhanh chóng các mô hình được huấn luyện trước đó trên văn bản cụ thể, điều chỉnh chúng trên tập dữ liệu của riêng bạn và sau đó chia sẻ chúng với cộng đồng trên [model hub](https://huggingface.co/models) của chúng tôi. Đồng thời, mỗi module python xác định một kiến trúc là hoàn toàn độc lập và có thể được sửa đổi để cho phép thực hiện nhanh các thí nghiệm nghiên cứu.
+
+🤗 Transformers được hỗ trợ bởi ba thư viện học sâu phổ biến nhất — [Jax](https://jax.readthedocs.io/en/latest/), [PyTorch](https://pytorch.org/) và [TensorFlow](https://www.tensorflow.org/) — với tích hợp mượt mà giữa chúng. Việc huấn luyện mô hình của bạn với một thư viện trước khi tải chúng để sử dụng trong suy luận với thư viện khác là rất dễ dàng.
+
+## Online demos
```
Comment (stevhliu) [inline README_vi.md:86]: Should this also be translated?

File: README_vi.md [hunk 2]
```diff
@@ -0,0 +1,579 @@
...
+Dòng code thứ hai tải xuống và lưu trữ bộ mô hình được huấn luyện được sử dụng bởi pipeline, trong khi dòng thứ ba đánh giá nó trên văn bản đã cho. Ở đây, câu trả lời là "tích cực" với độ tin cậy là 99,97%.
+
+Nhiều nhiệm vụ có sẵn một `pipeline` được huấn luyện trước, trong NLP nhưng cũng trong thị giác máy tính và giọng nói. Ví dụ, chúng ta có thể dễ dàng trích xuất các đối tượng được phát hiện trong một hình ảnh:
+
+``` python
+>>> import requests
+>>> from PIL import Image
+>>> from transformers import pipeline
+
+# Download an image with cute cats
```
Comment (stevhliu) [inline README_vi.md:163]: I noticed you translated `# Allocate a pipeline for sentiment-analysis` in the previous code snippet, so let's translate this as well for consistency (and other comments -like the one below - in the code examples)

**Review by stevhliu — APPROVED** [review]
Nice job!

I can't read any Vietnamese so unfortunately, I can't review the content itself. Do you know any other Vietnamese-readers in the ML community who would be able to review your work?

File: README_vi.md (hunk 1 above)
Comment (hoangsvit) [reply README_vi.md]: Yes, it needs to be translated.

File: README_vi.md (hunk 2 above)
Comment (hoangsvit) [reply README_vi.md]: I have updated it.

**Comment by hoangsvit** [comment]
> Nice job!
> 
> I can't read any Vietnamese so unfortunately, I can't review the content itself. Do you know any other Vietnamese-readers in the ML community who would be able to review your work?

Could you please help me, @davidnguyen-94 ?

**Comment by davidnguyen-94** [comment]
> > Nice job!
> > I can't read any Vietnamese so unfortunately, I can't review the content itself. Do you know any other Vietnamese-readers in the ML community who would be able to review your work?
> 
> Could you please help me, @davidnguyen-94 ?

Sure, I'd be happy to help!
I have checked and the translation is completely reasonable. @stevhliu @hoangsvit

**Pull request merged**